udemy_enroller_firefox.py

```diff
@@ -1,22 +1,23 @@
 # Install all the requirements by running requirements.py in IDLE or follow the alternate instructions at https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/
 # Make sure you have cleared all saved payment details on your Udemy account & the browser!
-# For firefox you need to manually install the driver on Arch Linux (sudo pacman -S geckodriver). Untested on other platforms.
 
-from bs4 import BeautifulSoup
+import time
+
 import requests
+from bs4 import BeautifulSoup
+from ruamel.yaml import YAML
 from selenium import webdriver
-from selenium.common.exceptions import TimeoutException
-from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import NoSuchElementException, TimeoutException
 from selenium.webdriver.common.by import By
-from selenium.common.exceptions import NoSuchElementException
-from webdriver_manager.firefox import GeckoDriverManager
-import time
-
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from webdriver_manager.microsoft import EdgeChromiumDriverManager

```

Comment:
Why did you replace GeckoDriverManager with EdgeChromiumDriverManager in firefox ? this will break firefox. Please fix. @ethankrein @aapatre